<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>839</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Dataset metadata</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_title">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>A descriptive title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Authors</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_authors">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Guybrush Threepwood, John Cleese</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>DOI</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_doi">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>only if published elsewhere (e.g. on figshare)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>References</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_references">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>publications (DOIs, arXiv, bioRxiv) or other links</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPlainTextEdit" name="plainTextEdit_notes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>400</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>General description.</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Tags</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_tags">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>e.g. HL60, cell-cycle</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>License</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="comboBox_license">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_8">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Circle</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="comboBox_circles">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_9">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Visibility</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="comboBox_vis">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Data files</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click &lt;span style=&quot; font-style:italic;&quot;&gt;Add data files&lt;/span&gt; below or drag&amp;amp;drop files to add resources (e.g. .rtdc files). Then, select one or multiple files to modify the supplementary resource metadata.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView_resources">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_add">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Add data files (resources)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_rem">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove selected resources</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_res_info">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Resource</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Filename</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLineEdit" name="lineEdit_res_filename">
                 <property name="acceptDrops">
                  <bool>false</bool>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_res_filename">
                 <property name="acceptDrops">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>undo filename changes</string>
                 </property>
                 <property name="text">
                  <string>r</string>
                 </property>
                 <property name="icon">
                  <iconset theme="undo">
                   <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Path</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_res_path">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_supplement" native="true">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Supplementary resource metadata</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="leftMargin">
                   <number>9</number>
                  </property>
                  <property name="rightMargin">
                   <number>9</number>
                  </property>
                  <property name="bottomMargin">
                   <number>6</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_14">
                    <property name="acceptDrops">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Preset</string>
                    </property>
                    <property name="buddy">
                     <cstring>comboBox_preset</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_preset">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="acceptDrops">
                     <bool>false</bool>
                    </property>
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_preset_store">
                    <property name="acceptDrops">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>store</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_preset_load">
                    <property name="acceptDrops">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>load</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="Line" name="line">
                  <property name="acceptDrops">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color:#888</string>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Plain</enum>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QScrollArea" name="scrollArea">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="acceptDrops">
                   <bool>false</bool>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Plain</enum>
                  </property>
                  <property name="lineWidth">
                   <number>0</number>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>509</width>
                     <height>557</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="SchemaWidget" name="widget_schema" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="acceptDrops">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SchemaWidget</class>
   <extends>QWidget</extends>
   <header>dcoraid.gui.upload.widget_schema</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
